integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java

```diff
@@ -927,32 +949,31 @@ private void changeConfigNegativeAuth(
       String effect, String configFile, String expectedErrorMsg, String username, String password)
       throws Exception {
     try {
-      HtmlPage page = submitConfigureForm(exporterUrl, effect, configFile, username, password);
+      final WebClient webClient = new WebClient();
+      setCredentials(webClient, username, password);
+      HtmlPage page = submitConfigureForm(exporterUrl, effect, configFile, webClient);
       throw new RuntimeException("Expected exception was not thrown ");
     } catch (FailingHttpStatusCodeException ex) {
       assertTrue((ex.getMessage()).contains(expectedErrorMsg));
     }
   }
 
-  private HtmlPage submitConfigureForm(
-      String exporterUrl, String effect, String configFile, String username, String password)
-      throws Exception {
-    final WebClient webClient = new WebClient();
-    setCredentials(webClient, username, password);
-    return submitConfigureForm(exporterUrl, effect, configFile, webClient);
-  }
-
   private HtmlPage submitConfigureForm(String exporterUrl, String effect, String configFile)
       throws Exception {
     final WebClient webClient = new WebClient();
+    webClient.getOptions().setThrowExceptionOnFailingStatusCode(false);
     setCredentials(webClient);
     return submitConfigureForm(exporterUrl, effect, configFile, webClient);
   }
 
   private HtmlPage submitConfigureForm(
       String exporterUrl, String effect, String configFile, WebClient webClient) throws Exception {
     // Get the first page
-    final HtmlPage page1 = webClient.getPage(exporterUrl);
+    HtmlPage page1 = webClient.getPage(exporterUrl);
```

Comment:
this seems odd - why only retry once? does this fail often?

Reply:
https://jira.oraclecorp.com/jira/browse/OWLS-80942

Reply:
it does not fail often, once in 2 weeks or so in the external Jen job, so added one more try if server is too busy